scout/parse/case.py

```diff
@@ -41,131 +46,87 @@ def get_correct_date(date_info):
     return datetime.datetime.now()
 
 
-def parse_case_data(
-    config=None,
-    ped=None,
-    owner=None,
-    vcf_snv=None,
-    vcf_sv=None,
-    vcf_cancer=None,
-    vcf_cancer_sv=None,
-    vcf_str=None,
-    smn_tsv=None,
-    peddy_ped=None,
-    peddy_sex=None,
-    peddy_check=None,
-    delivery_report=None,
-    multiqc=None,
-    cnv_report=None,
-    coverage_qc_report=None,
-    gene_fusion_report=None,
-    gene_fusion_report_research=None,
-):
+def parse_case_data(**kwargs):
     """Parse all data necessary for loading a case into scout
 
     This can be done either by providing a VCF file and other information
     on the command line. Or all the information can be specified in a config file.
     Please see Scout documentation for further instructions.
 
-    Args:
+    Possible keyword args:
+        cnv_report: Path to pdf file with CNV report
         config(dict): A yaml formatted config file
-        ped(iterable(str)): A ped formatted family file
+        coverage_qc_report: Path to html file with coverage and qc report
+        multiqc(str): Path to dir with multiqc information
         owner(str): The institute that owns a case
+        ped(iterable(str)): A ped formatted family file
+        peddy_ped(str): Path to a peddy ped
+        smn_tsv(str): Path to an SMN tsv file
+        vcf_cancer(str): Path to a vcf file
+        vcf_cancer_sv(str): Path to a vcf file
         vcf_snv(str): Path to a vcf file
         vcf_str(str): Path to a VCF file
         vcf_sv(str): Path to a vcf file
-        vcf_cancer(str): Path to a vcf file
-        vcf_cancer_sv(str): Path to a vcf file
-        smn_tsv(str): Path to an SMN tsv file
-        peddy_ped(str): Path to a peddy ped
-        multiqc(str): Path to dir with multiqc information
-        cnv_report: Path to pdf file with CNV report
-        coverage_qc_report: Path to html file with coverage and qc report
-        gene_fusion_report: Path to the gene fusions report
```

Comment:
These last 2 items in the list were removed and not ordered alphabetically in this comment